src/services/tours/tours-service.ts

```diff
@@ -23,14 +24,21 @@ export default class ToursService extends APIService {
     return this.handleTourListResult(result)
   }
 
-  public async findOne (id: string): Promise<SingleApiResponse<Tour>> {
+  public async findOne (id: string | undefined): Promise<SingleApiResponse<Tour>> {
+    const localTour = await this.localDatabaseService.getOne(id)
+    // no need to make an api call if the tour was only created locally
+    if (localTour?.status === TourStatusType.CREATED) {
+      const wrapper = this.getSuccessWrapper('got data from local database')
```

Comment:
@Saela Will these messages be removed before go-live?

If not, we should extract them to a centralized location and reference them from here.

Reply:
removed due to refactoring